Source/Header/DaySymbolsView.swift

```diff
@@ -26,13 +31,17 @@ class DaySymbolsView: UIView {
   }
 
   func configure() {
-
-    let daySymbols = NSCalendar.currentCalendar()
-      .shortWeekdaySymbols
+    let daySymbols = calendar.shortWeekdaySymbols
       .map { String($0.characters.first!)}
+    let weekendMask = [true] + [Bool](count: 5, repeatedValue: false) + [true]
+    var weekDays = Array(Zip2Sequence(daySymbols, weekendMask))
+
+    let offset = calendar.firstWeekday - 1
+    weekDays.shift(offset)
```

Comment:
The solution you've shown was used before I introduced tuple.
The need for `-1`  is because Apple uses 1 as a "Sunday"